src/Blocks/Rokalo.Blocks.Common.Extensions/ValidatorExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Rokalo.Blocks.Common.Extensions
+{
+    using FluentValidation;
+
+    public static class ValidatorExtensions
```

Comment:
put this file somewhere in Application Layer

Reply:
Done